lib/modules/dosomething/dosomething_global/dosomething_global.install

```diff
@@ -67,7 +67,7 @@ function dosomething_global_update_7003(&$sandbox) {
 /**
  * Uninstall some unused modules.
  */
-function dosomething_global_update_7004() {
+function dosomething_global_update_7005() {
```

Comment:
@mikecrittenden (Also sorry, I was scanning the file and didn't see that the long update for 7003 wasn't part of 7002. I guess I mean something like this for 7004 (since it has already run on staging): 

```
/**
 * Empty update function.
 */
function file_entity_update_7103() {
}
```